Draft-Accepted/RFCNNNN-Install-PowerShell.md

```diff
@@ -0,0 +1,205 @@
+some customers due to discoverability and some that have policies that don't allow for installing
+software onto Windows after initial provisioning.
+
+To compensate for these situations, customers will implement and use the "tools at hand" to perform
+management tasks. This leaves **Windows PowerShell 5.1**, not **PowerShell 7**, as the only
+automation solution available to many of them. While customers are missing out on useful new
+features that they may wish to have, they are certainly not receiving the performance and security
+improvements that come with each new release.
+
+This specification is an effort to help IT admins that wish to use new and updated releases of
+PowerShell 7. While this may not help customers that have post-deployment policy restrictions, it
+will help the majority of customers discover and update to the latest version of PowerShell for
+Windows.
+
+```output
+As an admin,
+I can use Windows PowerShell 5.1 to provision multiple hosts with PowerShell 7 over PSRemoting,
+so that I can leverage new capabilities in PS7.
+```
+
+```output
+As a user,
+I can discover and install PowerShell 7 from Windows PowerShell 5.1 on my Windows machine,
+so that I can leverage new capabilities in PS7.
+```
+
+Additional scenarios are enabled when considering a command versus a cmdlet. While Windows users
+would find benefit and familiarity to a cmdlet, a command opens up scenarios where Powershell is not
+pre-installed. Additionally, several installation issues due to open files are avoided by using a
+command and closing any open sessions of PowerShell.
+
+## Shipping in Windows
+
+There are several issues that prevent shipping PowerShell 7 in Windows;
+
+- Size
+- Support lifecycle differences
+
+### Size constraints
+
+ A complete installation of PowerShell 7 currently is around 200 MB (unpacked). This is due to
+ including everything (modules and .NET assemblies) to provide parity with what Windows PowerShell
+ 5.1 with .NET Framework provides. There is currently no plan by Windows or the .NET team to ship
+ newer .NET in Windows, so PS7 will need to continue to ship .NET with PS7.
+
+### Support lifecycle
+
+ Windows has a 5+5 year support lifecycle. .NET supports up to 3 years for Long Term Support (LTS)
+ versions. So even if size was not an issue, we cannot check in a version of PS7 with a .NET that is
+ not supported while Windows itself is supported.
+
+### Solution
+
+To solve these issues, the proposal is to ship a new command in Windows and the PowerShell Gallery
+that would;
+
+- Be locally discoverable
+- Perform a simple download and install of the latest LTS release of PowerShell
+- Include parameter options to select Stable,Preview or Long Term Service releases
+
+## Goals/Non-Goals
+
+Goals:
+
+- Be locally discoverable, not only an artifact on PSGallery.
+- By default, download and install the latest LTS release to the users CurrentUser Scope
```

Comment:
Where would PS7 install for CurrentUser scope? AppData? ProgramData?

If so, that adds complexity for the docs regarding `$PSHOME`.

Reply:
None of our install methods have a standard install for "CurrentUser". This tool would need to establish a new installation standard for "CurrentUser". We need this standardized so that we can document it.

Reply:
CurrentUser - $env:LOCALAPPDATA\Microsoft\pwsh